crates/walrus-service/src/node/event_blob_writer.rs

```diff
@@ -1439,7 +1456,11 @@ impl EventBlobWriter {
         let blob_id = *blob_metadata.blob_id();
 
         let mut batch = self.pending.batch();
-        match self.attest_blob(&blob_metadata, metadata.end).await {
+        let active_committees = self.node.committee_service.active_committees();
+        match self
+            .attest_blob(&blob_metadata, metadata.end, &active_committees)
+            .await
+        {
```

Comment:
Question/suggestion: Instead of getting the active committee first and passing it into the `attest_blob` function, wouldn't it be simpler to just get it inside the function?

Reply:
sure, will do